pkg/scheduler/schedulercache/cache.go

```diff
@@ -80,6 +80,33 @@ func newSchedulerCache(ttl, period time.Duration, stop <-chan struct{}) *schedul
 	}
 }
 
+// Snapshot takes a snapshot of the current schedulerCache.
```

Comment:
this has performance impact, now only used for debug purpose.

Reply:
Why don't add this NOTE in the comment here?

Reply:
agreed it would be nice to add in comment.